### PR TITLE
Bet

### PR DESCRIPTION
**Pull request by teddylo57**
-add functionality to receive a bet

### CODE REVIEW

File: backend/src/main/java/poker/enums/game/Action.java [hunk 1]
```diff
@@ -3,5 +3,8 @@
 public enum Action {
   FOLD,
   CHECK,
-  BET;
+  BET,
```
Comment (ericliu121187) [inline backend/src/main/java/poker/enums/game/Action.java]: Sooner or later, we'll want these two files to be synced up

https://github.com/ericliu121187/short-deck-poker/blob/master/frontend/src/constants/playerActionTypeEnum.js#L6

File: backend/src/main/java/poker/models/game/Player.java [hunk 2]
```diff
@@ -50,4 +52,12 @@ public void check() {
     this.toAct = false;
   }
 
+  public void bet(int betAmount) {
+    this.toAct = false;
+    Api.testTable.pot.addToPot(UserService.getPlayerFromUsername(username), betAmount);
+    Api.testTable.setCurrentRoundBetAmount(betAmount);
```
Comment (ericliu121187) [inline backend/src/main/java/poker/models/game/Player.java]: In the future, we may want to move these to hand instead of table.

Comment (teddylo57) [reply backend/src/main/java/poker/models/game/Player.java]: Sure, can you create a ticket for it?

File: backend/src/main/java/poker/enums/game/Action.java (hunk 1 above)
Comment (teddylo57) [reply backend/src/main/java/poker/enums/game/Action.java]: Synced

File: backend/src/main/java/poker/models/game/Player.java (hunk 2 above)
Comment (ericliu121187) [reply backend/src/main/java/poker/models/game/Player.java]: https://trello.com/c/clDYqSLQ/247-move-certain-fields-from-table-into-hand-model

**Pull request merged**